src/goto-instrument/accelerate/acceleration_utils.cpp

```diff
@@ -992,12 +992,10 @@ bool acceleration_utilst::do_nonrecursive(
     }
   }
 
-  for(expr_sett::iterator it=array_writes.begin();
-      it!=array_writes.end();
-      ++it)
+  for(auto it = array_writes.begin(); it != array_writes.end(); ++it)
   {
-    const exprt &lhs=*it;
-    const exprt &rhs=state[*it];
+    const auto &lhs = *it;
+    const auto &rhs = state[*it];
```

Comment:
How about changing this to a ranged for loop `for(const auto &lhs : array_writes)`?

Reply:
:+1:

Reply:
added